Code/Framework/AzToolsFramework/AzToolsFramework/aztoolsframework_files.cmake

```diff
@@ -562,6 +562,8 @@ set(FILES
     ViewportUi/ViewportUiCluster.cpp
     ViewportUi/ViewportUiSwitcher.h
     ViewportUi/ViewportUiSwitcher.cpp
+    ViewportUi/ViewportUiSwitcherManager.h
+    ViewportUi/ViewportUiSwitcherManager.cpp
```

Comment:
update: As discussed file naming should be more Component Mode focused for this (and moved to the Component Mode folder)